### PR TITLE
integrations: Add Screenshots & Notes for StatusPage.io.

### PR DESCRIPTION
**Pull request by MadElf1337**

### CODE REVIEW

File: webhook-integrations/StatusPage.io/notes.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+<br>
+StatusPage.io API: https://api.statuspage.io/v1/
+<br>
+(For StatusPage.io, API needs to configured first, & a key is required, which is obtained through the StatusPage.io
+account.)
+
+
+## Other Notes
+
+* No webhook payload was generated when a member was added/removed, but the app sent a message to the channel.
```
Comment (eeshangarg) [inline webhook-integrations/StatusPage.io/notes.md]: If the app sent a message to the channel, shouldn't a payload have been generated? Could you maybe look into if there is a way to configure that in StatusPage.io? Do you mean a member in Slack or a member in StatusPage.io? :)

Comment (MadElf1337) [reply webhook-integrations/StatusPage.io/notes.md]: @eeshangarg I tried everything to generate the webhook payload, it doesn't work

**Comment by MadElf1337** [comment]
@eeshangarg Done!
Can you review the task now?

**Comment by eeshangarg** [comment]
@MadElf1337: Awesome, this looks good! Merging! :)

**Pull request merged**